overwatch/parser.py

```diff
@@ -43,7 +43,8 @@ def extract_url_params(self, response):
 
     def extract_bio(self, response):
         content = response.find("li", class_="name").span
-        realname, age = re.search(GET_REAL_NAME_AND_AGE, content.text).groups()
+        _, realname, _, age = re.search(GET_REAL_NAME_AND_AGE, content.text).groups()
```

Comment:
You can use .group() function for eliminate the _ variables. Passing this param: 2, 4. e.g:

`name, age = re.search(GET_REAL_NAME_AND_AGE, content.text).group(2,4)`

Reply:
Cool ! i will do that